spec/apis/crypto_exchanges/coinbase_api*_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
```

Comment:
for all of these test files, you can remove the * from the name. The linter was just using * as a placeholder value.